src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        ColorSupplier supplier = new ColorSupplier();
+        double number = new Random().nextDouble(5);
+        Figure[] figures = new Figure[] {
+                new Circle(number, supplier.getRandomColor()),
+                new Rectangle(number, number, supplier.getRandomColor()),
+                new IsoscelesTrapezoid(number, number, number, supplier.getRandomColor()),
+                new RightTriangle(3, 4, supplier.getRandomColor()),
+                new Square(4, supplier.getRandomColor()),
+                new FigureSupplier().getDefaultFigure()
+        };
+
+        for (int i = 0; i < figures.length; i++) {
```

Comment:
In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters.
The other half of figures should have the same, default parameters.